docs/guides/issue_triage.md

```diff
@@ -0,0 +1,135 @@
+# Project issues
+
+General issues are listed at
+<https://github.com/project-chip/connectedhomeip/issues> .
+
+In order to be able to effectively follow up on these issues, they are separated
+into groups for further review and fixing based on current maintainers on
+specific areas of the code. The separation is done via `labels`.
+
+Issues that still need triaging are labeled as `needs triage`:
+
+<https://github.com/project-chip/connectedhomeip/issues?q=is%3Aissue+is%3Aopen+label%3A%22needs+triage%22>
+
+## Non-SDK issues
+
+In some cases the issue is integrating with ecosystems or environments are not
+supported. In these cases, issues can be re-directed to specific support pages
+rather than the SDK GitHub issues.
+
+### Google Integration
+
+- General integration link: <https://developers.home.google.com/matter>
+- Support via the communities areas:
+  - Stack overflow:
+        <https://stackoverflow.com/questions/tagged/google-smart-home>
+  - Google Nest Community:
+        <https://www.googlenestcommunity.com/t5/Smart-Home-Developer-Forum/bd-p/Smart-Home-Developer-Forum>
+
+### Environments not currently supported / not maintained
+
+Some items are explicitly not maintained currently
+
+- Windows integration: no official maintainer for this so currently not
+    supported
+- Old compiler support: CHIP requires a C++17 or higher compiler.
+
+## Platform maintainers
+
+This level of separation is generally for platform-specific issues (e.g. failure
+to commission for one specific platform, failure to run on some specific
+operating system).
+
+Contact is generally done on slack. E-mail addresses are not added here on
+purpose in order to avoid spam.
+
+| Platform     | Contact                                 | Label                                                                                                    | Note(s) |
+| ------------ | --------------------------------------- | -------------------------------------------------------------------------------------------------------- | ------- |
+| Android      | Andrei Litvin, Yunhan Wang, Yufeng Wang | [android](https://github.com/project-chip/connectedhomeip/issues?q=is%3Aopen+is%3Aissue+label%3Aandroid) |         |
```

Comment:
might be good to also add github handles here.

Reply:
I considered it, but thinking maybe as a followup once we find the syntax. It is also highly repetitive, so maybe we should have a separate index of name + handle. Generally was unsure how to have it, so made this first pass.